app/presenters/groups_presenter.rb

```diff
@@ -1,5 +1,5 @@
 class GroupsPresenter
-  def initialize(group: group, current_user: current_user)
+  def initialize(group:, current_user:)
```

Comment:
Yep. This is the ruby syntax for a required keyword argument. The key symbols are what correspond to the variable names within the body of `initialize`. The value expressions are what are evaluated as default values when nothing is passed. That is:

```
class Thing; end
def Thing.say(phrase: "Hi")
  puts phrase
end

Thing.say  #=> "Hi"
Thing.say(phrase: "Hello")  #=> "Hello"
```

That said, the value expressions `current_user` and `group` both evaluate to nil on line 2, so they're meaningless default values.  I removed them until we can come up with something that makes more sense.